jython/CreateSectionsFromBlocks.py

```diff
@@ -72,7 +72,7 @@ def getBlockEntryPointsList(b, entryPointList) :
 log.info( "CreateSectionsFromBlocks.py started" )
 sectionsCreated = 0 #count successes
 
-blocksSet = list(blocks.getNamedBeanSet()) # use a copy for the loop
+blocksSet = set(blocks.getNamedBeanSet()) # use a copy for the loop
```

Comment:
What is the purpose of list or set?  `blockSet = blocks.getNameBeanSet()` works.

Reply:
I'm changing the set, so I get ConcurrentCoModification error without making a copy.

Reply:
I understand the need for a copy.  I have never send list() or set() before.